config/parallel.json

```diff
@@ -0,0 +1,37 @@
+{
+    "capabilities": {
+        "bstack:options": {
+            "projectName": "Pytest Browserstack",
+            "buildName": "pytest-browserstack",
```

Comment:
Let's have `browserstack-build-1` (in all places in the repo) as the build name for sample repos - trying to keep it constant across sample repos..

Reply:
Done https://github.com/browserstack/pytest-browserstack/pull/2/commits/998555687793bd1c39a34317c2a5b3d86319041a